### PR TITLE
added missing props from Polyline & wrapped in declare module statement (react-native-maps)

### PR DESCRIPTION
**Pull request by methodbox**
There are two major issues currently with the `index.d.ts` type definitions in this module.

**Issue 1 - Your index.d.ts is not properly declared and so is not used.**

This can be fixed by wrapping the current contents of the file with a `declare module` statement like so:

```js
declare module "react-native-maps-directions" {
//  all the existing definitions
}
```
I have modified this in the PR and tested it with `react-native-maps` inside my app. There are no type conflicts with a basic config (with the except of #2).

**Issue 2 - There are a lot of missing props and type definitions required support your use of Polyline.**

This module inherently relies upon `react-native-maps` and imports and uses `Polyline` from that module, but these types are not inherited explicitly due to the fact that (per your own documentation) the `coordinates` prop is not acceptable or used on the `MapViewDirections` component.

> More props
Since the result rendered on screen is a MapView.Polyline component, all MapView.Polyline props – except for coordinates – are also accepted.

This means the types cannot be inherited, even in part, because `coordinates` is a required prop in the type definitions in `react-native-maps`.

In other words, they need to be defined explicitly in your `index.d.ts` within the `interface` called `MapViewDirectionsProps`.

I have done this here, based on their definitions from your own link to `react-native-maps`:  
https://github.com/react-native-community/react-native-maps/blob/master/docs/polyline.md#props

This fixes a number of issues, including a type error seen when using these props and effectively makes this package directly usable within Expo without type conflicts.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,39 +1,40 @@
-// Type definitions for react-native-maps-directions 1.6
-// Project: https://github.com/bramus/react-native-maps-directions
-// Definitions by: Ali Oguzhan Yildiz <https://github.com/alioguzhan>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.3
+declare module "react-native-maps-directions" {
```
Comment (methodbox) [inline index.d.ts]: Also wrapped in `declare module` statement.

**Comment by bramus** [comment]
Thanks for these additions! I've released a new beta version of 1.7.0.

Feel free to note down findings in https://github.com/bramus/react-native-maps-directions/issues/80

**Comment by methodbox** [comment]
> Thanks for these additions! I've released a new beta version of 1.7.0.
> 
> Feel free to note down findings in #80

No problem!

Sure, I'll test it out when I can. I'm working on some PRs for fixing related types conflicts with `react-native-maps`.
